.github/workflows/o3_schedule.yaml

```diff
@@ -6,7 +6,21 @@ on:
   schedule:
     - cron:  '0 6 * * MON'
 jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.3
+      - name: Get most recent image version
+        run: |
+          git clone https://github.com/rancher/rancher.git
+          cd rancher/
+          tag=$(git describe --tag $(git rev-list --tags --max-count=1))
```

Comment:
So this output is sorted so we're sure that the tag returned is really the latest version?

Reply:
Yes, exactly!